tests/console/zziplib.pm

```diff
@@ -36,7 +36,7 @@ sub run {
         zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => 'SDK';
     }
     # maintenance updates are registered with sdk module
-    elsif (get_var('FLAVOR') !~ /Updates|Incidents/) {
+    elsif (get_var('FLAVOR') !~ /Updates|Incidents/ || get_var('PUBLIC_CLOUD')) {
```

Comment:
Do the same also for `tests/console/soundtouch.pm & tests/console/wavpack.pm` ?

Reply:
there are two parts with this condition

Reply:
Fixed.